Makefile

```diff
@@ -185,13 +185,13 @@ docker-build-test: docker-clean-test docker-build-compile
 	  --attach stdout \
 	  --attach stderr \
 	  --rm \
-	  -w /go/src/github.com/pachyderm/pachyderm \
-	  -v $$GOPATH/src/github.com/pachyderm/pachyderm:/go/src/github.com/pachyderm/pachyderm \
+	  -w /pachyderm \
+	  -v $(PWD):/pachyderm \
```

Comment:
Just want to check that this make target has been tested or gets tested before merging, because I'm surprised this works. My expectation is that you'd need either `$$(PWD)` (make-escaped `$`) or `$(shell pwd)` or `$(CURDIR)` (but that would only work when running make from `$GOPATH/src/github.com/pachyderm/pachyderm`)

Reply:
Huh, yeah good catch. It works on my system, but then I'm on macOS and the case insensitivty allows even `$(Pwd)` to work. Doesn't TravisCI call this target at some point?